build.sbt

```diff
@@ -431,7 +454,7 @@ lazy val codegenCore =
     .settings(
       Compile / javacOptions ++= Seq("--release", "11"),
       Compile / scalacOptions ++= Seq("-release", "11"),
-      scalaVersion := Dependencies.ScalaVersionForDevTools)
```

Comment:
I renamed it to `tooling` to avoid confusion. DevTools is now also using Scala 2.13.